File: src/main/java/venda_montaria/tp1/Montaria.java
package venda_montaria.tp1;

public class Montaria extends Entidade {

	private String nome;
	private String raca;
	private String descricao;
	private String combustivel;
	private String raridade;
	private int capacidade;
	private float velocidade;
	static private int cont = 1;

	public static void setCont(int cont) {
		Montaria.cont = cont;
	}
	
	public Montaria(String nome, String raca, String descricao, String combustivel, String raridade, int capacidade,
			float velocidade) {
		setId(cont++);
		this.nome = nome;
		this.raca = raca;
		this.descricao = descricao;
		this.combustivel = combustivel;
		this.raridade = raridade;
		this.capacidade = capacidade;
		this.velocidade = velocidade;
	}
	public Montaria() {
		
	}

	public String getNome() {
		return nome;
	}

	public void setNome(String nome) {
		this.nome = nome;
	}

	public String getRaca() {
		return raca;
	}

	public void setRaca(String raca) {
		this.raca = raca;
	}

	public String getDescricao() {
		return descricao;
	}

	public void setDescricao(String descricao) {
		this.descricao = descricao;
	}

	public String getCombustivel() {
		return combustivel;
	}

	public void setCombustivel(String combustivel) {
		this.combustivel = combustivel;
	}

	public String getRaridade() {
		return raridade;
	}

	public void setRaridade(String raridade) {
		this.raridade = raridade;
	}

	public int getCapacidade() {
		return capacidade;
	}

	public void setCapacidade(int capacidade) {
		this.capacidade = capacidade;
	}

	public float getVelocidade() {
		return velocidade;
	}

	public void setVelocidade(float velocidade) {
		this.velocidade = velocidade;
	}

}

File: python/Montaria.py
from Entidade import Entidade

class Montaria (Entidade):
    __cont = 1
    def __init__(self, nome, raca, descricao, combustivel, raridade, capacidade, velocidade):
        self._setId(self.__cont)
        self.setCont(self.__cont+1)
        self.__nome = nome
        self.__raca = raca
        self.__descricao = descricao
        self.__combustivel = combustivel
        self.__raridade = raridade
        self.__capacidade = capacidade
        self.__velocidade = velocidade
    @classmethod
    def setCont(cls, cont):
        cls.__cont = cont

    def getNome(self):
        return self.__nome

    def setNome(self, nome):
        self.__nome = nome

    def getRaca(self):
        return self.__raca

    def setRaca(self, raca):
        self.__raca = raca

    def getDescricao(self):
        return self._descricao

    def setDescricao(self, descricao):
        self.descricao = descricao

    def getCombustivel(self):
        return self.__combustivel

    def setCombustivel(self, combustivel):
        self.combustivel = combustivel

    def getRaridade(self):
        return self.__raridade

    def setRaridade(self, raridade):
        self.__raridade = raridade

    def getCapacidade(self):
        return self.__capacidade

    def setCapacidade(self, capacidade):
        self.__capacidade = capacidade

    def getVelocidade(self):
        return self.__velocidade

    def setVelocidade(self, velocidade):
        self.__velocidade = velocidade